predictionprophet_deployment/agents/prophet_agent/deploy.py

```diff
@@ -40,51 +27,7 @@
 
 class DeployableTraderAgentER(DeployableTraderAgent):
     agent: AbstractBenchmarkedAgent
-    max_markets_per_run = 5
-
-    def recently_betted(self, market: AgentMarket) -> bool:
```

Comment:
This was all added to PMAT's DeployableAgent class, just forgot to remove it here.